invenio_config_tugraz/templates/security/email/welcome.html

```diff
@@ -0,0 +1,27 @@
+<p> 
+  {% trans email=user.email %}
+  Dear {{ email }}!
+  {% endtrans %}
+</p>
+
+<p>
+  {{ _('Welcome to TU Graz Repository!') }} <br>
+  {{ _('To help you get started, here are some useful links:') }}
+</p>
+
+<p>
+  <a href="{{ _('https://repository.tugraz.at/') }}"> {{ _('TU Graz Repository') }} </a> <br>
+  <a href="{{ _('https://repository.tugraz.at/static/documents/TUGraz_Repository_Guide_01_de.pdf') }}">{{ _('Reference Guide') }}</a><br>
+  <a href="{{ _('https://repository.tugraz.at/static/documents/TUGraz_Repository_Terms_And_Conditions_de.pdf') }}"> {{ _('Terms and Conditions') }} </a> <br>
```

Comment:
note: these hardcoded url should be replaced with something like:
 ```href="{{url_for('invenio_theme_tugraz.index')}}```, 
```href="{{url_for('invenio_theme_tugraz.guide')}} ``` and 
```href="{{url_for('invenio_theme_tugraz.terms')}}```

Reply:
For index page, absolutely. For the other two it might be good to have the documents in the respective language as well. Terms and Conditions is only available in German right now but there will probably be an English version as well, right?